lib/location/Kconfig

```diff
@@ -86,6 +86,11 @@ config LOCATION_METHOD_GNSS_PRIORITIZE_QZSS_ASSISTANCE
 	  needed at the same time. Enabling this option allows A-GNSS data request to be sent also
 	  when only QZSS assistance data (usually ephemerides) is needed.
 
+config LOCATION_SERVICE_NRF_CLOUD_GNSS_POS_CLOUD_SEND
```

Comment:
Would the name `LOCATION_SERVICE_NRF_CLOUD_GNSS_POS_SEND` be enough?

Reply:
Changed